src/main/java/org/fundacionjala/coding/mauricio/Isogram.java

```diff
@@ -0,0 +1,27 @@
+package org.fundacionjala.coding.mauricio;
+
+public final class Isogram {
+
+    private Isogram() {
+    }
+
+    public static boolean validateIsogram(final String word) {
+
+        boolean result = false;
+        if (word.length() == 0) {
+            result = true;
+        }
+
+        for (int i = 0; i < word.length(); i++) {
+            for (int j = 0; j < word.length(); j++) {
+                if ((word.charAt(i) == word.charAt(j)) && (i != j)) {
+                    result = false;
+                    return result;
+                } else {
```

Comment:
This `else` statement is no longer necessary, please consider removing it. So, you will have:
```java
        for (int i = 0; i < word.length(); i++) {
            for (int j = i + 1; j < word.length(); j++) {
                if (word.charAt(i) == word.charAt(j)) {
                    return false;
                }
            }
        }
```